azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/utilities/BottomCellViewHolder.kt

```diff
@@ -30,11 +30,19 @@ internal class BottomCellViewHolder(itemView: View) : RecyclerView.ViewHolder(it
 
     fun setCellData(bottomCellItem: BottomCellItem) {
         title.text = bottomCellItem.title
+        bottomCellItem.personaData?.let { personaData ->
```

Comment:
Can we do this instead:
`title.text = bottomCellItem.personaData?.renderedDisplayName ?: bottomCellItem.title`